src/styles/main.scss

```diff
@@ -1,3 +1,84 @@
+$day-size: 100px;
+$day-color: #eee;
+$calendar-padding: 10px;
+$day-gap: 1px;
+$day-names:
+  "mon" 0,
+  "tue" $day-size + $day-gap,
+  "wed" 2 * ($day-size + $day-gap),
+  "thu" 3 * ($day-size + $day-gap),
+  "fri" 4 * ($day-size + $day-gap),
+  "sat" 5 * ($day-size + $day-gap),
+  "sun" 6 * ($day-size + $day-gap);
+
+@mixin sqare($size) {
+  width: $size;
+  height: $size;
+  background-color: $day-color;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  padding: 100px;
+  margin: 0 auto;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: $calendar-padding;
+  margin: 0 auto;
+  box-sizing: border-box;
+  width: 6 * $day-gap + 7 * $day-size + 2 * $calendar-padding;
+  transition-duration: 0.5s;
+
+  &-day {
+    @include sqare($day-size);
+    box-sizing: border-box;
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
```suggestion
    justify-content: center;
    transition: color 0.5s, transform 0.5s;
  }
```

Reply:
you should specify properties directly, and in element styles for working in both directions: when `onHover`, and when cursor will be removed